### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by mattip**
- Add blurb about PyPy
- Change some examples of python use to python3.9
- Soften the commitment about latest Python versions
- Mention the commercial nature of the default channel

The last point could use some careful review: I am not sure I got the content and tone right.

### CODE REVIEW

**Comment by mattip** [comment]
Any thoughts on this PR?

**Comment by cjmartian** [comment]
@anaconda-issue-bot check

**Comment by jezdez** [comment]
@chenghlee Could you take a closer look at the documentation changes here if they match what Anaconda's policy is at the moment?

**Review by chenghlee — CHANGES_REQUESTED** [review]

File: docs/source/user-guide/tasks/manage-channels.rst
Comment (mattip) [reply docs/source/user-guide/tasks/manage-channels.rst:12]: changed to "and the various terms of service for their use"

File: docs/source/user-guide/install/download.rst
Comment (mattip) [reply docs/source/user-guide/install/download.rst]: adopted

File: docs/source/user-guide/concepts/channels.rst
Comment (mattip) [reply docs/source/user-guide/concepts/channels.rst:23]: adopted

**Comment by mattip** [comment]
changes made. CI is waiting for a maintainer to approve running

**Review by chenghlee — APPROVED** [review]

File: docs/source/user-guide/tasks/manage-python.rst
Comment (mattip) [inline docs/source/user-guide/tasks/manage-python.rst:93]: grr, typo. Will do another PR to fix it

**Pull request merged**
